nengo/utils/tests/test_testing.py

```diff
@@ -59,6 +55,31 @@ def test_analytics_norecord():
         analytics.get_filepath(ext='npz')
 
 
+def test_logger_record():
+    logger_obj = Logger('nengo.simulator.logs',
+                        'nengo.utils.tests.test_testing',
+                        'test_logger_record')
```

Comment:
Could we have a `logger` fixture that returns a logger configured like this (and with parametrized function names, too)?